frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/FeatureEnvironmentVariants.tsx

```diff
@@ -232,6 +235,8 @@ export const FeatureEnvironmentVariants = () => {
                 return (
                     <EnvironmentVariantsCard
                         key={environment.name}
+                        permission={UPDATE_FEATURE_ENVIRONMENT_VARIANTS}
```

Comment:
Not sure if it's worth passing the permission as prop instead of importing it inside the card component. WDYT?

Reply:
You are correct, sir, that is indeed overkill. I've removed the pass through and just imported it